github_ci_template.yml

```diff
@@ -27,10 +31,10 @@ Resources:
           Action: "sts:AssumeRole"
       Description: !Sub "DO NOT DELETE - Used by CloudFormation. Created by CloudFormation ${AWS::StackId}"
       ManagedPolicyArns:
-        #- !Sub "arn:${AWS::Partition}:iam::aws:policy/AdministratorAccess"
         - !Sub "arn:${AWS::Partition}:iam::aws:policy/AWSOrganizationsFullAccess"
         - !Sub "arn:${AWS::Partition}:iam::aws:policy/AWSCloudFormationFullAccess"
         - !Sub "arn:${AWS::Partition}:iam::aws:policy/IAMReadOnlyAccess"
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/AWSSSOReadOnly"
```

Comment:
Would it make more sense to use this policy instead of having the readonly SSO policy?

https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AWSSSOMasterAccountAdministrator.html